source/includes/api/_dataset.md

```diff
@@ -74,81 +76,140 @@ curl -X GET https://api.resourcewatch.org/v1/dataset
 
 ### Pagination
```

Comment:
This section may be worth moving to a more general place than under the `/dataset` endpoint specifically. @tiagojsag are our pagination mechanisms standardized enough to do that?

Reply:
Ideally yes, in practice, not sure. My suggestion is that we keep going through all MSs, copy-paste and validate as we go, and when we get to the end, if there's enough consistency, we refactor the docs (should be quick enough)

Reply:
Okay that sounds good to me. I'll add a task in JIRA so we don't forget.